task1.2/index.js

```diff
@@ -0,0 +1,24 @@
+import csv from 'csvtojson';
+import { createWriteStream } from 'fs';
+
+const csvFilePath = './csv/nodejs-hw1-ex1.csv';
+const txtFilePath = './txt/nodejs-hw1-ex2.txt';
+
+const writable = createWriteStream(txtFilePath, 'utf8');
+
+csv()
+  .fromFile(csvFilePath)
+  .subscribe((json) => new Promise((resolve) => resolve(json))
+    .then((csvLine) => {
+      writable.write(`${JSON.stringify(csvLine)}\n`);
+    })
+    .catch((error) => {
+      console.error(error);
+    })
+    .finally(() => {
+      console.log('Data read and written to file with or without error!');
```

Comment:
this log is not useful as it doesn't show what really happened (success or not?). I suggest to remove finally block and and logging to `then`